news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

```diff
@@ -0,0 +1,360 @@
+
+On 1 January 2020 I submitted [my first osu!taiko beatmap](https://osu.ppy.sh/beatmapsets/1087013), then I reached out to an osu!taiko NAT asking for beginner tips/resources, and they directed me to the [osu!taiko Modding & Mapping Hub Discord server](/wiki/Community/Discord_servers#mapping-and-modding). There, I've found a very welcoming community full of heartwarming people who were more than willing to help new people, which made me feel right at home. I started playing and mapping osu!taiko more and more, and eventually, I got hooked.
+
+Nowadays, while my monthly playcount is low, I still play osu! and osu!taiko pretty equally.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What is your source of motivation to stay involved in basically all aspects of osu! even after all these years?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>
+
+the osu! community has been, and still is, a once in a lifetime experience for me, and I've grown attached to all the workflows and people I've met along the way. The reason I'm still here, and will be for the foreseeable future, is because it's a place that I can call *home* in some form, despite the ups and downs.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/avatars/MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What would you call the greatest gift that osu! has given you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/14102976" style="background-image: url('/wiki/shared/avatars/Hivie.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-nat" href="https://osu.ppy.sh/users/14102976">Hivie</a></p>
+
+I want to name 2 things:
+
+1. People skills: I've been a pretty timid person for a good chunk for my life, and osu! has helped me develop my social skills in a way I didn't expect, both through working together on projects, and through managing and leading teams.
```

Comment:
```suggestion
1. Personal development: I've been a pretty timid person for a good chunk for my life, and osu! has helped me develop my social skills in a way I didn't expect, both through working together on projects, and through managing and leading teams.
```
"people skills" looks odd to me :thinking:

Reply:
Would prefer to keep it as is since it's a real term https://en.wikipedia.org/wiki/People_skills